### ISSUE TITLE
Problem with adding the tifffile.c extension

### ISSUE THREAD

**Issue by unidesigner** [imageio/imageio#222]
What is the proper way to build and add the Tifffile.c extension to the imageio package?
I can download http://www.lfd.uci.edu/~gohlke/code/tifffile.c and build it with the provided setup.py, but putting the generated _tiffile.so into the imageio.plugins directory causes problems with the existing _tifffile.py file. The exception is
```
python2.7/site-packages/imageio/plugins/tifffile.pyc in _open(self, bigtiff, byteorder, software)
    207             if not _tifffile:
    208                 load_lib()
--> 209             self._tf = _tifffile.TiffWriter(self.request.get_local_filename(),
    210                                             bigtiff, byteorder, software)
    211             self._meta = {}
AttributeError: 'module' object has no attribute 'TiffWriter'
```
Thanks for help of doing this properly.

**Comment by almarklein** [comment]
IIUC it should be possible to install tiffile seperately, because the plugin will attempt to import the global tiffile module first: https://github.com/imageio/imageio/blob/master/imageio/plugins/tifffile.py#L18-L24

@blink1073 might have more to add; I don't know anything about the tiffile c extension.

**Comment by blink1073** [comment]
The python and c files are versioned together.  It looks like I need to cut a new release of the tifffile package https://github.com/blink1073/tifffile/issues/19.  If that is installed, imageio will prefer it over its own local version.

**Comment by blink1073** [comment]
I just released v0.12 of tifffile with the updates.

**Comment by unidesigner** [comment]
Thanks! It works now using tifffile directly. Would be nice to have it integrated in imageio at some point.
